test/traversal_tests.html

```diff
@@ -0,0 +1,4 @@
+<div id="1">
```

Comment:
Can you rename this to `traversal-tests.html` (with a dash)?

This dir has a mix of files with dashes and underscores but I'd like to standardize on one and eventually update them all to match. Since there are more dash filenames overall in the project, let's prefer that for new files.